packages/common/CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## Unreleased
+
+### Added
+
+New consensus endpoints:
+- `getBakerEarliestWinTime`
+- `getBlockCertificates`
+- `getBakersRewardPeriod`
+- `getWinningBakersEpoch`
+- `getFirstBlockEpoch`
+
+### Breaking changes
+
+- `commissionRates` is now added to the `getPoolInfo` under `bakerPoolStatus.PoolInfo`
+
```

Comment:
Damn, I must have missed that when doing the protocol version 6 type changes.. I know this is essentially breaking, but since this is basically something that should have been there all along, should we maybe document this as a "fix" instead?

Users will likely not be constructing objects of this type; merely reading from the as they are returned from the GRPC client, so I think it's fair to do this. What do you think?

Reply:
No, this is only going to be there in 6.1 node. This field is not present in version 6 node API.

Reply:
Alright, that makes sense. I the logic of how users will be interacting with this object still stands, especially when it's also a change done in a minor release of the node. Maybe just put it under "Added"?

Reply:
I agree about the logic.

It's kind of an odd situation. From the node's perspective this is a "conservative" extension of the API, so a minor version bump makes sense.